ui/core/proto_utils/equipped_item.ts

```diff
@@ -473,17 +479,17 @@ export class EquippedItem {
 		}
 	}
 
-	// Whether this item could have an extra socket, assuming Blacksmithing.
+	// Whether this item could have an extra socket
 	couldHaveExtraSocket(): boolean {
-		return [ItemType.ItemTypeWaist, ItemType.ItemTypeWrist, ItemType.ItemTypeHands].includes(this.item.type);
+		return [ItemType.ItemTypeWrist, ItemType.ItemTypeHands].includes(this.item.type);
 	}
 
 	requiresExtraSocket(): boolean {
 		return [ItemType.ItemTypeWrist, ItemType.ItemTypeHands].includes(this.item.type) && this.hasExtraGem() && this._gems[this._gems.length - 1] != null;
 	}
```

Comment:
Replace `[ItemType.ItemTypeWrist, ItemType.ItemTypeHands].includes(this.item.type)` with `this.couldHaveExtraSocket()` for cleanliness.